Nos/Models/CoreData/Event+CoreDataClass.swift

```diff
@@ -915,7 +915,11 @@ public class Event: NosManagedObject {
             }
             try? context.saveIfNeeded()
             let tags = note.allTags as? [[String]] ?? []
-            return NoteParser.parse(content: content, tags: tags, context: context)
+            return NoteParser().parse(
```

Comment:
Not 100% sure how practical it is to test this, but... I'd probably make a `NoteParser` parameter for this func, even if it just defaults to `NoteParser()`. That would make it testable.